Composer/packages/client/src/pages/setting/app-settings/AppSettings.tsx

```diff
@@ -50,15 +50,37 @@ const AppSettings: React.FC<RouteComponentProps> = () => {
 
   const renderElectronSettings = isElectron();
 
-  const languageOptions = [{ key: 'en-US', text: formatMessage('English (US)') }];
+  const languageOptions = [
+    { key: 'en-US', text: 'English (US)', englishName: 'English (US)' },
+    { key: 'cs', text: 'Čeština‎ Czech', englishName: 'Czech' },
+    { key: 'de', text: 'Deutsch‎', englishName: 'German' },
+    { key: 'es', text: 'Español‎', englishName: 'Spanish' },
+    { key: 'fr', text: 'Français‎', englishName: 'French' },
+    { key: 'hu', text: 'Magyar‎', englishName: 'Hungarian' },
+    { key: 'it', text: 'Italiano‎', englishName: ' Italian' },
+    { key: 'ja', text: '日本語‎', englishName: 'Japanese' },
+    { key: 'ko', text: '한국어‎', englishName: 'Korean' },
+    { key: 'nl', text: 'Nederlands‎', englishName: 'Dutch (Netherlands)' },
+    { key: 'pl', text: 'Polski‎', englishName: 'Polish' },
+    { key: 'pt-BR', text: 'Português (Brasil)‎', englishName: 'Portuguese (Brazil)' },
+    { key: 'pt-PT', text: 'Português (Portugal)‎', englishName: 'Portuguese (Portugal)' },
+    { key: 'ru', text: 'Русский‎', englishName: 'Russian' },
+    { key: 'sv', text: 'Svenska‎', englishName: 'Swedish' },
+    { key: 'tr', text: 'Türkçe‎', englishName: 'Turkish' },
+    { key: 'zh-Hans', text: '中文(简体)‎', englishName: 'Chinese(Simplified)' },
+    { key: 'hz-Hant', text: '中文(繁體)', englishName: 'Chinese(Traditional)' },
+  ];
+
   if (process.env.NODE_ENV !== 'production') {
     languageOptions.push({
       key: 'en-US-pseudo',
       text: formatMessage('Pseudo'),
+      englishName: 'pseudo',
     });
     languageOptions.push({
       key: 'en-US-DoesNotExist',
       text: formatMessage('Does Not Exist'),
+      englishName: 'does not exist',
```

Comment:
why do we need a "Does Not Exits"?

Reply:
I think does not exist is there for testing when the language selected is not found.  Pseudo and Does Not Exist only show up in dev mode.